src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -882,7 +890,7 @@ internal override bool HasComplete(CompletionPart part)
             return _state.HasComplete(part);
         }
 
-        internal override void ForceComplete(SourceLocation locationOpt, CancellationToken cancellationToken)
+        internal override void ForceComplete([NotNull] SourceLocation? locationOpt, CancellationToken cancellationToken)
```

Comment:
>[NotNull] [](start = 45, length = 9)

How?

Reply:
Well this one just made no sense. 😆 Now fixed.